src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/DoNotDecreaseInheritedMemberVisibilityTests.Fixer.cs

```diff
@@ -12,22 +11,387 @@ public class DoNotDecreaseInheritedMemberVisibilityFixerTests : CodeFixTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicDoNotDecreaseInheritedMemberVisibilityAnalyzer();
+            return new DoNotDecreaseInheritedMemberVisibilityAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpDoNotDecreaseInheritedMemberVisibilityAnalyzer();
+            return new DoNotDecreaseInheritedMemberVisibilityAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicDoNotDecreaseInheritedMemberVisibilityFixer();
+            return new DoNotDecreaseInheritedMemberVisibilityFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpDoNotDecreaseInheritedMemberVisibilityFixer();
+            return new DoNotDecreaseInheritedMemberVisibilityFixer();
+        }
+
+        [Fact]
+        public void DecreaseCSharpMemberVisibility()
```

Comment:
I've changed these test names to have the language first since it sounds like that's the more common pattern.